presto-array/src/main/java/com/facebook/presto/array/IntBigArray.java

```diff
@@ -142,4 +142,9 @@ private void allocateNewSegment()
         capacity += SEGMENT_SIZE;
         segments++;
     }
+
```

Comment:
Is this whole commit ("Import quicksort") 100% copy/paste? Or did you modify there something? If there were modifications, please split this into two commits: copy/paste and your modifications.